README.md

```diff
@@ -73,7 +73,7 @@ the GitHub [release page](https://github.com/kong/deck/releases)
 or install by downloading the binary:
 
 ```shell
-$ curl -sL https://github.com/kong/deck/releases/download/v1.40.0/deck_1.40.0_linux_amd64.tar.gz -o deck.tar.gz
```

Comment:
I believe we should change those links at some point so that they do not need updating on each release (e.g using latest release link and not adding the version information to the binary file names)

Reply:
Good idea! I also want to update the release process such that `homebrew/deck` doesn't require manual merge each time. I will keep this in mind too. Thanks!